### PR TITLE
Xedra new spell Heal Fatigue

### PR DESCRIPTION
**Pull request by KymasTuran**
#### Summary
Content "New spell Heal Fatigue"

#### Purpose of change
I think its a "must have"

#### Describe the solution
Just a new spell, a bit harder to find, as a long casting cost.

#### Describe alternatives you've considered
A potion or dream coffee. Or a dream coffee maker that consume dreams to make supercoffee.

#### Testing
Find or spawn the spell, study it, cast. Not for combat.

#### Additional context
Perhaps spells learned in dreams, and/or wake up after a nightmare summon a shadow. This mod as many possibilities.

### CODE REVIEW

**Comment by KymasTuran** [comment]
There is an extra file that I cant remove. 
This site as a nasty learning curve.

**Review by GuardianDll — CHANGES_REQUESTED** [review]
Nope, that's a bad pr, it removes few bugfixes and edit things in aftershock (the spell itself is good, you just need to add an artifact into a loot group). Try again. If you have problems with github, welcome to our discord, i kindly guide you how to do it properly.

**Comment by Maleclypse** [comment]
Oh I do like this.  I hope you can get the github part fixed, I still have a number of PRs I need to finish.

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json [hunk 1]
```diff
@@ -73,5 +73,20 @@
     "symbol": "O",
     "color": "white_magenta",
     "use_action": { "type": "learn_spell", "spells": [ "self_healing" ] }
+  },
+  {
+    "id": "strange_coffee",
+    "type": "GENERIC",
+    "category": "artifacts",
+    "name": { "str": "strange_coffee", "str_pl": "strange_coffee" },
+    "description": "A small jar with coffee, it is moving in weird patterns even without being touch. After watching the movement of the dark liquid for some time you are sure that repeat a certain pattern, like a message",
```
Comment (MNG-cataclysm) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json:82]:
```suggestion
    "description": "A small jar with coffee, moving in weird patterns with no apparent cause.  After watching the swirls for some time you are sure that it loops back on it's movements, like some sort of message.",
```

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]

**Comment by KymasTuran** [comment]
> Nope, that's a bad pr, it removes few bugfixes and edit things in aftershock (the spell itself is good, you just need to add an artifact into a loot group). Try again. If you have problems with github, welcome to our discord, i kindly guide you how to do it properly.

The Aftershock one are not supposed to be here. 
Thank you for your kind offer.

**Pull request closed without merging**